app/views/application/_tag.html.erb

```diff
@@ -1,5 +1,14 @@
-<% if (tag == selected) %>
-  <%=link_to tag.name, @library, {class: "badge rounded-pill bg-success tag", data: {bulk_item_tags: "#{model_id}"} } %>
-<% else %>
-  <%=link_to tag.name, [@library, tag: tag.name], {class: "badge rounded-pill bg-secondary tag", data: {bulk_item_tags: "#{model_id}"}} %>
-<% end %>
+<% 
+  paramtag = [*params[:tag]]
```

Comment:
This could be extracted into a method in application_helper.rb, perhaps, keep the view code a bit tidier?

Reply:
Makes sense.   I think I learned about helpers after I already had this written.